### PR TITLE
added button component

### PR DESCRIPTION
**Pull request by tusharkandpal**
- different types: primary, outline, disabled, icon, group, float, link
- styles for the different types

### CODE REVIEW

**Review by Bharatrathod-81 — APPROVED** [review]
Perfect

**Review by ZuberDunge — APPROVED** [review]
Looks good to me

**Review by Bharatrathod-81 — COMMENTED** [review]
good work

File: components/button/button.css [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+}
+.btn-sm {
+  padding: 0.5rem 1rem;
+}
+.btn-md {
+  padding: 0.5rem 2rem;
+}
+.btn-lg {
+  padding: 0.8rem 3.5rem;
+}
```
Comment (SaurabhChawda) [inline components/button/button.css:33]: use Pixel for less than 1rem.

**Review by SaurabhChawda — APPROVED** [review]

File: components/button/button.css (hunk 1 above)
Comment (tusharkandpal) [reply components/button/button.css:33]: I've done this in too many places to change now. 😬

**Pull request merged**